client/watch.py

```diff
@@ -43,7 +45,7 @@ async def watch():
             yield WatchRequest(create_request=req)
 
             while not self.is_cancel:
-                await asyncio.sleep(0.5)
+                await asyncio.sleep(0.2)
```

Comment:
How did we arrive at this 0.2?

Reply:
I accidentally changed it during testing, and in fact we don't need to change it.